chemper/graphs/cluster_graph.py

```diff
@@ -415,9 +415,9 @@ def add_mol(self, mol, smirks_atoms_list):
         Parameters
         ----------
         mol: chemper Mol object
-        smirks_atoms_list: list of dicts
-            This is a list of dictionaries of the form [{smirks index: atom index}]
-            each atom (by index) in the dictionary will be added the relevant
+        smirks_atoms_list: list of tuples
+            This is a list of tuples with atom indices [ (indices), (...]
+            each atom (by index) in the tuple will be added the relevant
             AtomStorage by smirks index
```

Comment:
Fix formatting for the [ (indices) ... ] section.
Do you mean each atom will be _assigned_ the relevant AtomStorage?